src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -6318,303 +6364,327 @@ namespace EnergyPlus::OutputReportTabular {
-                            }
-                            // CR7783 fix
-                            if (UtilityRoutines::SameString(curUnits, "kWh/s")) {
-                                curUnits = "W";
-                                curConversionFactor *= 3600000.0;
-                            }
-                            if (UtilityRoutines::SameString(curUnits, "GJ/s")) {
-                                curUnits = "kW";
-                                curConversionFactor *= 1000000.0;
-                            }
-                            if (UtilityRoutines::SameString(curUnits, "MJ/s")) {
-                                curUnits = "kW";
-                                curConversionFactor *= 1000.0;
-                            }
-                            if (UtilityRoutines::SameString(curUnits, "therm/s")) {
-                                curUnits = "kBtu/h";
-                                curConversionFactor *= 360000.0;
-                            }
-                            if (UtilityRoutines::SameString(curUnits, "kBtu/s")) {
-                                curUnits = "kBtu/h";
-                                curConversionFactor *= 3600.0;
-                            }
-                            if (UtilityRoutines::SameString(curUnits, "ton-hrs/s")) {
-                                curUnits = "ton";
-                                curConversionFactor *= 3600.0;
-                            }
-                            columnHead(columnRecount - 1) = ort->MonthlyColumns(curCol).varName + curAggString + '[' + curUnits + ']';
-                            columnHead(columnRecount) = ort->MonthlyColumns(curCol).varName + " {TIMESTAMP} ";
-                            minVal = storedMaxVal;
-                            maxVal = storedMinVal;
-                            for (lMonth = 1; lMonth <= 12; ++lMonth) {
-                                if (ort->IsMonthGathered(lMonth)) {
-                                    curVal = ort->MonthlyColumns(curCol).reslt(lMonth);
-                                    // CR7788 the conversion factors were causing an overflow for the InchPound case since the
-                                    // value was very small
-                                    // restructured the following lines to hide showing HUGE and -HUGE values in output table CR8154 Glazer
-                                    if ((curVal < veryLarge) && (curVal > verySmall)) {
-                                        curVal = curVal * curConversionFactor + curConversionOffset;
+                            } else if (SELECT_CASE_var == iAggType::ValueWhenMaxMin) {
+                                ++columnRecount;
+                                if (ort->MonthlyColumns(curCol).avgSum == OutputProcessor::StoreType::Summed) {
+                                    curUnits += "/s";
+                                }
+                                if (UtilityRoutines::SameString(curUnits, "J/s")) {
```

Comment:
At some point I feel like it would be worth trying to move these sorts of things into a nice constexpr-ish map where the unit string is a key.  Then we can rely on the compiler to do optimized lookups and storage and we don't have to have these big if blocks anymore.

Reply:
Thanks! So this will be left unchanged for now?

Reply:
Yeah it's ok for now.